packages/react-console/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "4.59.1",
-    "@patternfly/react-core": "^4.75.9",
+    "@patternfly/react-core": "^4.75.10",
```

Comment:
If @redallen is ok with bumping `@patternfly/react-core` here, then I'm good too.

Reply:
Yeah, this should have been done in an earlier PR.